frontend/src/components/PReadsNavbar.js

```diff
@@ -40,6 +47,11 @@ export default class PReadsNavbar extends React.Component {
                   </NavItem>
                 );
               })}
+              {this.isLoggedIn() && (
+                <NavItem>
+                  <NavLink href={'/logout'}>Logout</NavLink>
+                </NavItem>
+              )}
```

Comment:
Could we also make this navigate to Login when user is not logged in?

In future we might direct this to a profile instead, or (if doable) a dropdown that they can then choose the option to logout... But that's future work